setup.sh

```diff
@@ -67,13 +78,19 @@ function updateenv() {
         ${PYTHON} -m pip install --upgrade cython
     else
         # Is not Raspberry
-        read -p "Do you want to install hyperopt dependencies [y/N]? "
+        if [[ "$1" == "auto" ]]
+        then
+          REPLY="y"
+        else
+          read -p "Do you want to install hyperopt dependencies [y/N]? "
+        fi
```

Comment:
Rather than having all these conditionals, it might be better to put this code into a function

```
${PYTHON} -m pip install --upgrade -r ${REQUIREMENTS} ${REQUIREMENTS_HYPEROPT} ${REQUIREMENTS_PLOT}
    if [ $? -ne 0 ]; then
        echo "Failed installing dependencies"
        exit 1
    fi
    ${PYTHON} -m pip install -e .
    if [ $? -ne 0 ]; then
        echo "Failed installing Freqtrade"
        exit 1
    fi
    echo "pip install completed"
    echo
```

and then if someone selects `auto-install`, just set

```
        REQUIREMENTS=requirements.txt
        REQUIREMENTS_PLOT="-r requirements-plot.txt"
        REQUIREMENTS_HYPEROPT="-r requirements-hyperopt.txt"
```

and then call that function